solr_plugins/Makefile

```diff
@@ -1,4 +1,4 @@
-SUBDIRS = Schlagwortfolgensuche Bibelstellensuche PrefixSortFacetten
+SUBDIRS = Schlagwortfolgensuche Bibelstellensuche PrefixSortFacetten MultiLanguageDocTransformer MultiLanguageQueryParser
```

Comment:
I don't think variable assignment in shell scripts works with spaces around the assignment operator. :-(  (I wish it did!)